packages/create-request-form/src/utils/formatAddress.ts

```diff
@@ -0,0 +1,11 @@
+export const formatAddress = (
+  address: string,
+  first: number = 6,
+  last: number = 4
+): string => {
+  if (address?.length < first + last) {
+    throw new Error("Address too short to format properly.");
+  }
```

Comment:
This logic seems odd. Why would address length be compared with first/last name?

Reply:
Oh okay, it's not first/last name. It's first and last chars. Please change var name.